core/deployment/src/main/java/io/quarkus/deployment/IsDockerWorking.java

```diff
@@ -48,12 +51,6 @@ public boolean getAsBoolean() {
         return false;
     }
 
-    public static class IsDockerRunningSilent extends IsDockerWorking {
```

Comment:
I think this and the constructor above that has been removed is really part of the API, and can't really just be removed.

Reply:
👍